spec/models/notification_spec.rb

```diff
@@ -630,137 +630,6 @@ def create(**opts)
     end
   end
 
-  describe "#recent_report" do
```

Comment:
Per the comment above, I think we still need this `#recent_report` method, so let's restore the test as well.